diff --git a/common/include/x86emitter/x86_intrin.h b/common/include/x86emitter/x86_intrin.h
index 78c68f3f3..7b87912b5 100644
--- a/common/include/x86emitter/x86_intrin.h
+++ b/common/include/x86emitter/x86_intrin.h
@@ -50,7 +50,7 @@ static __inline__ __attribute__((always_inline)) void cpuid(int CPUInfo[], const
     __cpuid(InfoType, CPUInfo[0], CPUInfo[1], CPUInfo[2], CPUInfo[3]);
 }
 
-#if defined(__clang__) || __GNUC__ < 8 || (__GNUC__ == 8 && __GNUC_MINOR__ < 2)
+#if defined(__clang__) || __GNUC__ < 10 || (__GNUC__ == 1 && __GNUC_MINOR__ < 3)
 // _xgetbv on gcc 8.1 is broken (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=85684).
 // It also isn't present on clang and earlier versions of gcc.
 static __inline__ __attribute__((always_inline)) unsigned long long xgetbv(unsigned int index)

